.github/workflows/deploy.yml

```
@@ -0,0 +1,102 @@
+        region: [europe-west4, europe-west3]
+    steps:
+      - name: "Auth with Google Cloud"
+        uses: 'google-github-actions/auth@v2'
+        with:
+          credentials_json: ${{ secrets.SERVICE_ACCOUNT_KEY }}
+
+      - name: "Deploy Migrations to Cloud Run"
+        if: ${{ matrix.region == 'europe-west4' }}
+        uses: 'google-github-actions/deploy-cloudrun@v2'
+        with:
+          image: europe-docker.pkg.dev/ghost-activitypub/activitypub/migrations:${{ needs.build-test-push.outputs.migrations_docker_version }}
```

actionlint (1.7.4)

property "build-test-push" is not defined in object type {}

(expression)